packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -255,6 +259,20 @@ const View = BaseView.extend({
     return true;
   },
 
+  /**
+   * Should the security events be displayed ?
+   *
+   * @returns {Boolean}
+   * @private
+   */
+  _areSecurityEventsVisible() {
+    if (this.window.location.href.includes('security_events=true')) {
```

Comment:
Lets move this check into the broker fetch function, https://github.com/mozilla/fxa/blob/52c86a8d17d17e6362bac4b5a3e0f1b85954015b/packages/fxa-content-server/app/scripts/models/auth_brokers/base.js#L150,

From there you can do something like
```
this.setCapability('showSecurityEvents', !! this.getSearchParam('security_events'));
```

This will remove the need to do `this.window.location.href.includes`. From the settings page, you will be able to directly check the broker capabilities and decided to show the link.